### PR TITLE
feat: add company name to summary page

### PR DESCRIPTION
**Pull request by akaJSaunders**
### JIRA link

https://companieshouse.atlassian.net/browse/LFA-1660

### Change description

Screenshot 2020-06-04 at 12 43 58

Screenshot 2020-06-05 at 10 22 27

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are commited to keeping commit history clean, consistent and linear. To achive this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/views/confirmation.njk [hunk 1]
```diff
@@ -31,6 +31,19 @@
         <a href="javascript:if(window.print)window.print()" class="govuk-link">Print a copy of this appeal</a>
       </p>
       <div class="govuk-!-display-none-screen">
+        <h2 class="govuk-heading-l">Penalty Details</h2>
```
Comment (n-r-b) [inline src/views/confirmation.njk:34]: Is it part of the ticket to add this to the confirmation page?

File: test/controllers/PenaltyDetailsController.test.ts [hunk 2]
```diff
@@ -82,8 +85,16 @@ describe('PenaltyDetailsController', () => {
                 }
             });
 
+            companyProfileService.getCompanyProfile('SC123123').resolves({
```
Comment (n-r-b) [inline test/controllers/PenaltyDetailsController.test.ts:88]: It's great that we have a test for the happy path here, but what happens when this service returns an error?

File: src/views/confirmation.njk (hunk 1 above)
Comment (akaJSaunders) [reply src/views/confirmation.njk:34]: This is for the printout, which is part of the ticket

File: test/controllers/PenaltyDetailsController.test.ts (hunk 2 above)
Comment (akaJSaunders) [reply test/controllers/PenaltyDetailsController.test.ts:88]: well spotted

File: src/views/confirmation.njk (hunk 1 above)
Comment (n-r-b) [reply src/views/confirmation.njk:34]: ok, just thought i'd check

**Comment by akaJSaunders** [comment]
I have added a screenshot of how the print at confirmation page looks like with the latest commit

**Pull request merged**